examples/mpu6050/main.go

```diff
@@ -11,12 +11,24 @@ import (
 func main() {
 	machine.I2C0.Configure(machine.I2CConfig{})
 
-	accel := mpu6050.New(machine.I2C0)
-	accel.Configure()
+	mpuDevice := mpu6050.New(machine.I2C0)
+	mpuDevice.Init(mpu6050.IMUConfig{
+		AccRange:  mpu6050.ACCEL_RANGE_16,
+		GyroRange: mpu6050.GYRO_RANGE_2000,
+	})
 
 	for {
-		x, y, z := accel.ReadAcceleration()
-		println(x, y, z)
 		time.Sleep(time.Millisecond * 100)
+		err := mpuDevice.Update()
```

Comment:
I believe `Update` should have a parameter with the sensor values it should update?

Reply:
Yes, we should probably solve #345 before merging

Reply:
I'd leave this for last since I still don't have a good rebase methodology. As soon as everything else is reviewed and OK I'll try my hand at rebasing one more time